### ISSUE TITLE
.cpp:863: virtual nvinfer1::ILayer* nvinfer1::Network::getLayer(int) const: Assertion `layerIndex >= 0' failed.

### ISSUE THREAD

**Issue by Bob-jpg** [deepcam-cn/yolov5-face#101]
运行环境
![图片](https://user-images.githubusercontent.com/56429665/144813277-c99b4a88-7212-4958-9897-71cd5192b572.png)
![图片](https://user-images.githubusercontent.com/56429665/144813362-6c767f27-a140-4aa1-8934-b189e1081696.png)

**Comment by Bob-jpg** [comment]
![图片](https://user-images.githubusercontent.com/56429665/144813560-eadde579-ab09-4b15-ad63-070aba20a39d.png)
this is my pip list ,but I use the main.py,something is wrong

**Comment by bobo0810** [comment]
> ![图片](https://user-images.githubusercontent.com/56429665/144813560-eadde579-ab09-4b15-ad63-070aba20a39d.png) this is my pip list ,but I use the main.py,something is wrong

Try to ensure that the tensorRT version is consistent with the official version

**Comment by Bob-jpg** [comment]
> 
yeah ,TensorRT-6.0.1.5.Ubuntu-18.04.x86_64-gnu.cuda-10.1.cudnn7.6.tar  this is my tensorrt version

**Comment by Bob-jpg** [comment]
> Try to ensure that the tensorRT version is consistent with the official version
![图片](https://user-images.githubusercontent.com/56429665/144815470-846ade1c-cf38-42ad-b4d7-fac51a9d44a0.png)
this is my cuda and cudnn version

**Comment by bobo0810** [comment]
TensorRT is written differently for each large version, V8 is written differently.  try tensorrt7.2

**Comment by Bob-jpg** [comment]
OK，thank you  *-*
